ocs_ci/utility/utils.py

```diff
@@ -458,6 +458,60 @@ def run_cmd(cmd, secrets=None, timeout=600, ignore_error=False, **kwargs):
     return mask_secrets(completed_process.stdout.decode(), secrets)
 
 
+def run_cmd_multicluster(cmd, secrets=None, timeout=600, ignore_error=False, **kwargs):
+    """
+    Run command on multiple clusters. Useful in multicluster scenarios
+    This is wrapper around exec_cmd
+
+    Args:
+        cmd (str): command to be run
+        secrets (list): A list of secrets to be masked with asterisks
+            This kwarg is popped in order to not interfere with
+            subprocess.run(``**kwargs``)
+        timeout (int): Timeout for the command, defaults to 600 seconds.
+        ignore_error (bool): True if ignore non zero return code and do not
+            raise the exception.
+
+    Raises:
+        CommandFailed: In case the command execution fails
+
+    Returns:
+        list : of CompletedProcess objects as per cluster's index in config.clusters
+            i.e. [cluster1_completedprocess, None, cluster2_completedprocess]
+            if command execution skipped on a particular cluster then corresponding entry will have null
+
+    """
+    # Skip indexed cluster while running commands
+    # Useful to skip operations on ACM cluster
+    skip_index = kwargs.get("skip_index", None)
```

Comment:
I think that skip_index should be regular parameter of the function. I think it should not be then passed to exec_cmd as if you have it in kwargs and do not pop that parameter from kwargs it will be passed to exec_cmd.

Reply:
done